features/other_b/series.feature

```diff
@@ -209,21 +213,50 @@ Feature: Create and Edit Series
     Then I should not see the image "title" text "Restricted" within "h2"
 
   Scenario: View user's series index
-
     Given I am logged in as "whoever" with password "whatever"
       And I add the work "grumble" to series "polarbears"
     When I go to whoever's series page
     Then I should see "whoever's Series"
       And I should see "polarbears"
 
   Scenario: Series index for maaany series
-    Given I am logged in as "whoever" with password "whatever"  
+    Given I am logged in as "whoever" with password "whatever"
       And I add the work "grumble" to "32" series "penguins"
     When I go to whoever's series page
     Then I should see "penguins30"
     When I follow "Next"
     Then I should see "penguins0"
-  
+
+  Scenario: Removing self as co-author from co-authored series
```

Comment:
This is good! However, I think we can shorten it a bit (and make it faster) by taking advantage of some of our more flexible step definitions and only checking the most necessary bits:

```
    Given I am logged in as "sun"
      And I set up the draft "Sweetie Bell" as part of a series "Ponies"
      And I add the co-author "moon"
      And I post the work without preview
    When I view the series "Ponies"
      And I follow "Remove Me As Author"
    Then I should see "You have been removed as an author from the series and its works."
      And I should see "by moon"
      And I should not see "by moon, sun"
    When I go to my works page
    Then I should not see "Sweetie Bell"
```